tests/testthat/test_aggregate_sex.R

```diff
@@ -0,0 +1,51 @@
+testthat::context("aggregate_sex() tests")
```

Comment:
From this article (https://www.tidyverse.org/blog/2019/04/testthat-2-1-0/), should we not use `context()`?

Reply:
I'm ok with that. Is there something we should include instead?

Reply:
Good catch Spencer, that makes sense to me. I had an older version of testthat so just had to update.